onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1128,6 +1128,16 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
                "2D tensor with shape (max_sequence_length, head_size / 2).",
                "T",
                OpSchema::Optional)
+        .Input(9,
+               "pos_ids",
+               "2D tensor with shape (batch_size, sequence_length).",
+               "tensor(int64)",
+               OpSchema::Optional)
+        .Input(10,
+               "attention_mask",
```

Comment:
Suggest to rename it to attention_bias like https://github.com/microsoft/onnxruntime/blob/5bd31636d5b1bbd25e772b5c9b678956a1636617/onnxruntime/core/graph/contrib_ops/bert_defs.cc#L423-L427

attention mask is usually 0/1 value. Your use case is an bias tensor that added to QK.